Nautilus/Handlers/CraftDataHandler.cs

```diff
@@ -27,10 +27,11 @@ public static void AddToGroup(TechGroup group, TechCategory category, TechType t
     /// <param name="group">The TechGroup you want to add your TechType to.</param>
     /// <param name="category">The TechCategory (in the TechGroup) you want to add your TechType to.</param>
     /// <param name="techType">The TechType you want to add.</param>
-    /// <param name="after">Added TechType will be added after this TechType, for sorting purposes.</param>
-    public static void AddToGroup(TechGroup group, TechCategory category, TechType techType, TechType after)
+    /// <param name="after">Whether to add after or insert before the target, for sorting purposes.</param>
+    /// <param name="target">It will be added/inserted next to this item or at the end/beginning if not found.</param>
+    public static void AddToGroup(TechGroup group, TechCategory category, TechType techType, bool after = true, TechType target = TechType.None)
```

Comment:
Please mark any breaking changes in your PR description.

Reply:
This breaking change can actually be avoided by swapping the last two params. Shall I do that?

Reply:
I think it would still break, since the optional parameters must stay the same as well.

Reply:
I'm not sure many people used this override anyway, a breaking change would really be fine. It's just helpful when the breaking changes are listed in the PR description.

Reply:
By swapping the last two params (both of which are optional) it averts the breaking change while keeping the new functionality.